src/main/java/io/wispforest/owo/network/OwoNetChannel.java

```diff
@@ -445,27 +445,6 @@ private <R extends Record> void createEndec(Class<R> messageClass, int handlerIn
         }
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
🤨

Reply:
this method was fully unused, and had an EndecBuffer#write without context that I found while searching for all uses of it

Reply:
Well then